localstack/services/apigateway/helpers.py

```diff
@@ -108,6 +106,78 @@ def __init__(self):
         self.client_certificates = {}
 
 
+class Resolver:
```

Comment:
This is a pretty neat utility class! 👍 Should we add a couple of small unit tests, to cover the most basic functionality? (and maybe also some edge cases, like unresolvable references, ...)